javascript/testing_javascript/project_testing_practice.md

```diff
@@ -3,7 +3,7 @@ Let's practice!  This testing thing really is not that difficult, but it _is_ qu
 
 ### Using ES6 import statements with Jest
 
-By default, the current version of Jest will not recognize ES6 import statements. In order for you to be able to use ES6 modules for this project you may follow the Jest instructions that are laid out [here](https://jestjs.io/docs/en/getting-started#using-babel). It should take less then two minutes.
+By default, the current version of Jest will not recognize ES6 import statements. In order for you to be able to use ES6 modules for this project you may follow the [Jest instructions for using Babel](https://jestjs.io/docs/en/getting-started#using-babel). It should take less than two minutes.
```

Comment:
```suggestion
By default, the current version of Jest will not recognize ES6 import statements. In order for you to be able to use ES6 modules for this project you may follow the [Jest instructions for using Babel](https://jestjs.io/docs/en/getting-started#using-babel).
```
Not a blocker, but it'd be better to remove phrasing like "It should take less than two minutes" - it's unnecessary and may make a learner feel bad about not being able to complete something in time.

Reply:
I agree this can be removed